.github/workflows/backend-ci-cd.yml

```
@@ -0,0 +1,102 @@
+          restore-keys: |
+            ${{ runner.os }}-m2-
+      - name: Run Checkstyle for ${{ matrix.service }}
+        working-directory: apps/backend/${{ matrix.service }}
+        run: |
+          mvn checkstyle:check
+  test-analyze-build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        service: [
+          'discovery-server','gateway', 'config-server', 'user-service', 'file-service', 'sync-service', 'common' ]
```

YAMLlint (1.35.1)

[error] too few spaces after comma

(commas)